actioncable/app/assets/javascripts/actioncable.esm.js

```diff
@@ -255,7 +262,12 @@ Connection.prototype.events = {
 
      case message_types.confirmation:
       this.subscriptions.confirmSubscription(identifier);
-      return this.subscriptions.notify(identifier, "connected");
+      if (this.reconnectAttempted) {
+        this.reconnectAttempted = false;
+        return this.subscriptions.notify(identifier, "reconnected");
```

Comment:
Isn't there a backward compatibility issue here? Some people may be listening to `connected` and their callback won't be invoked anymore.

Would it be possible to either trigger both, or to pass wether we are connected or reconnect as an argument?

Reply:
Yes indeed. I wasn't sure which approach to adopt.

I guess that I should trigger both.

Reply:
I updated the code to trigger both callbacks.

Reply:
It's ok. What do you have in mind?

Reply:
You want to use the `connected()` callback with a `{reconnected: true}` argument?

Reply:
You want to use the `connected()` callback with a `{reconnect: true}` argument ?
```suggestion
        if (this.reconnectAttempted) {
          this.reconnectAttempted = false
          return this.subscriptions.notify(identifier, "connected", {reconnected: true})
        } else {
          return this.subscriptions.notify(identifier, "connected", {reconnected: false})
        }
```

Reply:
Something like that yes. I see that `notify` supports passing extra data, so that should be easy. 

First step would be to look if other events have arguments like this, and if some do, then think of what would be the most consistent wya to pass it.

If this is the first one, then what you suggest looks good to me.

Reply:
That's the first one. The `received()` callback has an argument but is used for a different purpose.

Reply:
I updated the PR.